app/services/DepartureService.scala

```diff
@@ -46,7 +46,7 @@ class DepartureService @Inject()(departureIdRepository: DepartureIdRepository)(i
       xmlMessage <- updateMesSenMES3(departureId, messageCorrelationId)
     } yield MessageWithStatus(messageId, dateTime, messageType, xmlMessage, SubmissionPending, messageCorrelationId)
 
-  def createDeparture(eori: String, nodeSeq: NodeSeq, channelType: ChannelType, boxOpt: Option[Box]): Future[ParseHandler[Departure]] =
+  def createDeparture(enrolmentId: Ior[TURN, EORINumber], nodeSeq: NodeSeq, channelType: ChannelType, boxOpt: Option[Box]): Future[ParseHandler[Departure]] =
```

Comment:
What is an Ior?

Reply:
It's like an Igor but less scary.  Or it could be an inclusive or.  Or it could be both.

Reply:
What @crystalmark said - it can be either Left or Right or Both. This is great for our case because we can have either the old enrolment, the new enrolment or both, but None would be an unauthenticated user so we don't want that.
https://typelevel.org/cats/datatypes/ior.html